pyworld/pyworld.pyi

```diff
@@ -0,0 +1,399 @@
+from typing import Optional, Tuple
```

Comment:
`typing.Tuple` is Deprecated.
However, removing this requires Python 3.9 (or maybe Python 3.7).
https://docs.python.org/3.13/library/typing.html#typing.Tuple